Handle Masking and substitution for NaN & FILLVAL Values (#92)
swxsoc/swxsoc: docs/user-guide/reading_writing_data.rst

This function returns the full :py:class:`~pathlib.Path` to the CDF file that was generated.
From this you can validate and distribute your CDF file.

> Missing data, ``NaN``, masks, and FILLVAL on write/read are described in :doc:`fillval_and_masks`.

Reviewer: ```suggestion
How to deal with missing or bad data is :doc:`fillval_and_masks`.
```
